### PR TITLE
Update WordPlate date

### PR DESCRIPTION
**Pull request by vinkla**
- [X] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [x] Suggesting edits to an existing Topic page

***********EDITING AN EXISTING TOPIC PAGE************

I'm suggesting these edits to an existing topic:
- [X] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

I've updated to the date for WordPlate to its initial commit.

### CODE REVIEW

File: topics/wordplate/index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ created_by: Vincent Klaiber
 display_name: WordPlate
 github_url: https://github.com/wordplate
 logo: wordplate.png
-released: March 18, 2015
+released: October 4, 2013
```
Comment (cheshire137) [inline topics/wordplate/index.md]: If this is the initial commit date, does it make sense for this field? API.md describes `released` as being the date of first release.

Comment (vinkla) [reply topics/wordplate/index.md]: It depends. March 18, 2015 was the day we created a tag and submitted it to Packagist. Before that it was a boilerplate without tags.

Comment (cheshire137) [reply topics/wordplate/index.md]: Hmm, was it publicly available October 4, 2013 and in a working state? Even if it wasn't on Packagist at that date, if it was at least an alpha that others could use, that seems fine.

Comment (vinkla) [reply topics/wordplate/index.md]: Yes, it was working fine on October 4, 2013.

**Pull request merged**